app/views/comments/turbo/_tabs.html.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+  <ul class="nav nav-pills">
+    <li class="nav-item <%= @formType == "horizontal" ? "active" : "" %>" role="presentation">
```

Comment:
How about using action_name to check the active status instead of creating instance variable @formType in the controller?